src/azure-cli/azure/cli/command_modules/apim/custom.py

```diff
@@ -952,3 +954,104 @@ def apim_ds_purge(client, service_name, location, no_wait=False):
         client.deleted_services.begin_purge,
         service_name=service_name,
         location=location)
+
+# Graphql Resolver Operations
+
+
+def apim_graphql_resolver_create(
+        client, resource_group_name, service_name, api_id, resolver_id, display_name,
+        path, description=None, no_wait=False):
+    """Creates a new Resolver. """
+    parameters = ResolverContract(
+        display_name=display_name,
+        path=path,
+        description=description
+    )
+
+    return sdk_no_wait(no_wait, client.graph_ql_api_resolver.create_or_update,
+                       resource_group_name=resource_group_name,
+                       service_name=service_name,
+                       api_id=api_id,
+                       resolver_id=resolver_id,
+                       parameters=parameters)
+
+
+def apim_graphql_resolver_delete(
+        client, resource_group_name, service_name, api_id, resolver_id, no_wait=False, if_match=None):
+
+    return sdk_no_wait(no_wait, client.graph_ql_api_resolver.delete,
+                       resource_group_name=resource_group_name,
+                       service_name=service_name,
+                       api_id=api_id,
+                       resolver_id=resolver_id,
+                       if_match="*" if if_match is None else if_match)
+
+
+def apim_graphql_resolver_show(client, resource_group_name, service_name, api_id, resolver_id):
+    
+        return client.graph_ql_api_resolver.get(
+            resource_group_name=resource_group_name,
+            service_name=service_name,
+            api_id=api_id,
+            resolver_id=resolver_id)
```

Comment:
Since there is no special logic in this custom method, I believe you can directly call the SDK method.
```suggestion
```